rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_21.json

```diff
@@ -0,0 +1,3109 @@
+                                                    "maximum_outdoor_airflow": 3750,
+                                                    "air_energy_recovery": {
+                                                        "id": "Air Energy Recovery 1",
+                                                        "energy_recovery_operation": "WHEN_MINIMUM_OUTSIDE_AIR"
+                                                    }
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "boilers": [
+                            {
+                                "id": "Boiler 1",
+                                "loop": "Boiler Loop 1",
+                                "energy_source_type": "NATURAL_GAS"
+                            }
+                        ],
+                        "chillers": [
+                            {
+                                "id": "Chiller 1",
+                                "cooling_loop": "Chiller Loop 1"
+                            }
+                        ],
+                        "pumps": [
+                            {
+                                "id": "Boiler Pump 1",
+                                "loop_or_piping": "Boiler Loop 1",
+                                "speed_control": "FIXED_SPEED"
+                            },
+                            {
+                                "id": "Chiller Pump 1",
+                                "loop_or_piping": "Chiller Loop 1",
+                                "speed_control": "FIXED_SPEED"
+                            },
+                            {
+                                "id": "Secondary CHW Pump",
+                                "loop_or_piping": "Secondary CHW Loop 1",
+                                "speed_control": "VARIABLE_SPEED"
+                            }
+                        ],
+                        "fluid_loops": [
+                            {
+                                "id": "Boiler Loop 1",
+                                "type": "HEATING"
+                            },
+                            {
+                                "id": "Chiller Loop 1",
+                                "type": "COOLING",
+                                "child_loops": [
+                                    {
+                                        "id": "Secondary CHW Loop 1"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "weather": {
+                    "climate_zone": "CZ5A"
+                }
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
```

Comment:
Could we have ```weather``` and ```climate_zone``` keys (here and also in the following TCDs?

Reply:
the weather and climate_zone are on lines 307 & 308. They are showing up as the last objects within the baseline/proposed RMDs